tests/main/hook-permissions/test-snap/meta/hooks/post-refresh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if snapctl is-connected upower-observe; then
+    dbus-send --print-reply --system --dest=org.freedesktop.UPower /org/freedesktop/UPower org.freedesktop.UPower.EnumerateDevices > "$SNAP_COMMON/enumerate.txt" 2>&1
```

Comment:
super minor nitpick, but I think this would read easier with it split up into multiple lines like:

```suggestion
    dbus-send \
      --print-reply \
      --system \
      --dest=org.freedesktop.UPower \
      /org/freedesktop/UPower \
      org.freedesktop.UPower.EnumerateDevices \
         > "$SNAP_COMMON/enumerate.txt" 2>&1
```